db/carrier_icons.yml

```diff
@@ -2,11 +2,52 @@
   name: ups
   label: UPS
   group: other
+-
+  name: usps
+  label: USPS
+  group: other
+-
+  name: sendle
+  label: Sendle
+  group: other
+  
+-
+  name: poste-italiane
+  label: PosteItaliane
+  group: other
+-
+  name: mondial-relay
+  label: MondialRelay
+  group: other
 -
   name: fedex
-  label: FedEx
+  label: Fedex
+  group: other
+-
+  name: evri
+  label: Evri
+  group: other
+-
+  name: dpd
+  label: Dpd
   group: other
 -
   name: dhl
   label: DHL
   group: other
+-
+  name: correos
+  label: Correos
+  group: other
+-
+  name: colissimo
```

Comment:
Could you create a colissimo_symbol.svg version: https://www.colissimo.entreprise.laposte.fr/en/tools-and-services/communication-kit

![CleanShot 2024-02-02 at 12 35 35@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/9a4da96b-059a-4be0-b637-24e6c8b12dc0)

Reply:
![CleanShot 2024-02-02 at 12 39 54@2x](https://github.com/Checkout-Blocks/shipping_carrier_icons/assets/3484527/4ae3d5d0-a1c6-4892-b388-5d937dfbeef4)